node_modules/sorted-union-stream/index.js

```diff
@@ -33,8 +33,9 @@ var union = function (streamA, streamB, toKey) {
         var keyB = toKey(dataB)
 
         if (keyA === keyB) {
```

Comment:
Can you elaborate on what's going on here? And what this being fixed looks like?